README.md

```diff
@@ -90,13 +90,65 @@
 ## Getting Started
 
 
-### Prerequisites
-
-
-
-### Installation
-
-
+### Setup
+#### First you need is load Alpine.js and Tailwindcss
+
+```bash
+<script src="https://unpkg.com/alpinejs" defer></script>
+```
+
+```bash
+<script src="https://cdn.tailwindcss.com"></script>
+```
+
+#### Than you must add Auth Modal Library
+
+```bash 
+<script src="https://cdn.jsdelivr.net/gh/tringug/auth-modal/src/auth-modal.js" defer></script> 
+```
+
+#### And you must add Adapter SDK (You can use what you won´t)
+
+For example we use <a href="https://appwrite.io/">Appwrite</a>
+
+```bash
+<script src="https://cdn.jsdelivr.net/npm/appwrite@7.0.0"></script>
```

Comment:
V tomto momente uz som celkom zmateny. Neviem ci to davat do head, do body.. na zaciatok, na koniec.. pod seba, do seba, v akom podari.

Skusme to nejako lepsie vysvetlit, asi uz v uvode setupu.